scripts/encode_data.py

```diff
@@ -96,27 +96,35 @@ def parse_args(test: ArgList = None) -> argparse.Namespace:
       help='''Number of processes to use.
           (default: the number of CPUs in the system)''',
       default=None)
+  parser.add_argument(
+      '--scale',
+      type=int,
+      help='''Weight scale for the entrties. The value should be a unsigned
```

Comment:
s/entrties/entries/ ?
```suggestion
      help='''Weight scale for the entries. The value should be a unsigned
```

Reply:
Good catch! Thanks.